packages/functionals/botpress-qna/src/storage.js

```diff
@@ -47,7 +47,7 @@ export default class Storage {
     }
   }
 
-  async saveQuestion(data, id = null) {
+  async saveQuestion(data, id = null, syncNlu = true) {
```

Comment:
NIT for future reference, in the majority of cases we should avoid adding boolean flags as it's unclear what they do from reading the consumer code.
More explicit and self-documenting approach is to have an options object, would be `async saveQuestion(data, id = null, { syncNlu = true } = {})` in this case (could also move `id` inside of it)

Reply:
Yeah, I thought about it.